resources/views/privacy-interception.blade.php

```diff
@@ -21,15 +21,15 @@
                                 {!! __('privacy.not-signed-yet') !!}
                             </p>
                         </div>
-                    @elseif($user->privacy_ack_at < $agreement->valid_at)
+                    @elseif($user->privacy_ack_at->isBefore($agreement->valid_at))
                         <div class="card mb-3">
                             <p class="card-body mb-0">
                                 {!! __('privacy.we-changed') !!}
                             </p>
                         </div>
                     @endif
 
-                    @if($user->privacy_ack_at < $agreement->valid_at || $user->privacy_ack_at == null)
+                    @if($user->privacy_ack_at == null || $agreement->valid_at->isAfter($user->privacy_ack_at))
```

Comment:
Could also be shortened to `@if($agreement->valid_at->isAfter($user->privacy_ack_at))`

Again test first please

Reply:
Nope, because privacy_ack_at is null at default.